kubernetes/cleanup

```diff
@@ -8,5 +8,6 @@ log "Cleaning up old containers"
 for i in `docker ps -a | grep k8s | awk '{print $1}'`; do docker rm -vf $i; done
 
 log "Cleaning the tmp and manifests folders"
-sudo rm -rf tmp/* manifests/* 
+sudo rm -rf tmp/* manifests/*
+sudo rm -rf /var/log/pods/*
```

Comment:
i was looking at `systemd-exec` or `systemd-run` for this script so that you can add on-the-fly arguments to your `kubelet` and wanted to bring it up, but it seemed overkill to me. this way you can redirect the logs to syslog i think. i think, not 100% sure :). do you guys think it would be overkill?